agents/node/vcxagent-core/vcx-agent.js

```diff
@@ -23,12 +37,35 @@ async function createVcxAgent ({ agentName, protocolType, agencyUrl, seed, webho
     connections[connectionName] = connection
   }
 
-  async function initVcx (name = agentName) {
-    logger.info(`Initializing VCX of ${name}`)
+  /**
+   * Initializes setting in memory, open pool, open wallet, set webhook url if present in agent provison
```

Comment:
Setting what in memory? "... initilizes and opens wallet and pool, and sets webhook url in agency if present in config.`

Reply:
updated